test/schema.json

```
@@ -0,0 +1,475 @@
+          }
+        }
+      ]
+    },
+    {
+      "$comment": "Only one assertion is required. It doesn't matter which type.",
+      "anyOf": [
+        {
+          "properties": {
+            "defaultTestParams": {
+              "$ref": "#/$defs/anyExp"
+            }
+          }
+        },
+        {
+          "properties": {
+            "tests": {
+              "type": "array",
+              "items": {
+                "$ref": "#/$defs/anyExp"
+              }
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "$defs": {
+    "defaultTestParams": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "testType": {
+          "$ref": "#/$defs/testType"
+        },
+        "locale": {
+          "$ref": "#/$defs/locale"
+        },
+        "src": {
+          "$ref": "#/$defs/src"
+        },
+        "params": {
+          "$ref": "#/$defs/params"
+        },
+        "exp": {
+          "$ref": "#/$defs/exp"
+        },
+        "expCleanSrc": {
+          "$ref": "#/$defs/expCleanSrc"
+        },
+        "expParts": {
+          "$ref": "#/$defs/expParts"
+        },
+        "expErrors": {
+          "$ref": "#/$defs/expErrors"
+        }
+      }
+    },
+    "test": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "description": {
+          "$ref": "#/$defs/description"
+        },
+        "testType": {
+          "$ref": "#/$defs/testType"
+        },
+        "locale": {
+          "$ref": "#/$defs/locale"
+        },
+        "src": {
+          "$ref": "#/$defs/src"
+        },
+        "params": {
+          "$ref": "#/$defs/params"
+        },
+        "exp": {
+          "$ref": "#/$defs/exp"
+        },
+        "expCleanSrc": {
+          "$ref": "#/$defs/expCleanSrc"
+        },
+        "expParts": {
+          "$ref": "#/$defs/expParts"
+        },
+        "expErrors": {
+          "$ref": "#/$defs/expErrors"
+        },
+        "only": {
+          "type": "boolean",
+          "description": "Normally not set. A flag to use during development to only run one or more specific tests."
+        }
+      }
+    },
+    "description": {
+      "type": "string",
+      "description": "Information about the test."
+    },
+    "testType": {
+      "description": "The category of test.",
+      "enum": [
+        "syntax",
+        "function"
+      ]
+    },
+    "locale": {
+      "description": "The locale to use for formatting.",
+      "type": "string"
+    },
+    "src": {
+      "description": "The MF2 syntax source.",
+      "type": "string"
+    },
+    "params": {
+      "description": "Parameters to pass in to the formatter for resolving external variables.",
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/var"
+      }
+    },
+    "var": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "type",
+        "value"
+      ],
```

The type should be optional, as JSON already provides an implicit type that covers all the cases except for `"datetime"`.

This change will also need to be reflected in the `oneOf` array below.

```suggestion
      "required": [
        "value"
      ],
```